apps/webcert/src/components/utils/ExpandableText.tsx

```diff
@@ -1,52 +1,42 @@
-import type React from 'react'
-import { useEffect, useState } from 'react'
+import { useState } from 'react'
 import styled from 'styled-components'
 import type { FlattenSimpleInterpolation } from 'styled-components/macro'
 import { ChevronDownIcon, ChevronUpIcon } from '../../images'
 import { sanitizeText } from '../../utils'
 
-const StyledLink = styled.a`
+const StyledLink = styled.button`
   display: flex;
-  align-items: bottom;
+  align-items: center;
 `
 
-interface Props {
-  text: string
+export function ExpandableText({
+  text,
+  maxLength,
+  additionalStyles,
+}: {
+  text?: string
   maxLength: number
   additionalStyles?: FlattenSimpleInterpolation
-}
-
-export const ExpandableText: React.FC<Props> = ({ text, maxLength, additionalStyles }) => {
-  const [expand, setExpand] = useState(false)
-
-  useEffect(() => {
-    setExpand(false)
-  }, [text])
+}) {
+  const isExpandable = text && text.length > maxLength
+  const [expand, setExpand] = useState(isExpandable ? false : true)
```

Comment:
This could be simplified

```const [expand, setExpand] = useState(!isExpandable)```

If u feel that this makes it less readable, ignore this comment.

Reply:
Good call, i'll fix this